src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -105,7 +142,14 @@
                                         (if (> num-value current-limit-amount)
                                           (reset! input-value (str current-limit-amount))
                                           (reset! input-value v))
-                                        (reagent/flush))))]
+                                        (reagent/flush))))
```

Comment:
@J-Son89 Just wondering if we really need the existing `reagent/flush` calls?

I tried removing them and I didn't find issues, but I didn't test it deeply.